electron/run.js

```diff
@@ -104,15 +116,25 @@ function onData(phase, data) {
 		container.hidden = false
 	} else if (phase === 'jml-output') {
 		let container = document.querySelector('#jml-container')
-		let serialize = data => JSON.stringify(data, null, 2)
-		let ress = serialize(data.results)
+
 		document
 			.querySelector('#distributions')
 			.appendChild(makeTableFromObject(data.distributions))
+
 		document
 			.querySelector('#probabilities')
 			.appendChild(makeTableFromObject(data.probabilities))
-		document.querySelector('#results').innerHTML = `<pre>${ress}</pre>`
+
+		data.results = data.results.map(item => {
+			if (item.Probability < 0.05) {
```

Comment:
Capital P?

Reply:
Oh, JML output. NVM

Reply:
Yeah, that's what the column is called, so that's what we get as our object key.